src/parser/permit-generation-module.ts

```diff
@@ -26,6 +26,7 @@ import envConfigSchema, { EnvConfigType, envValidator } from "../types/env-type"
 import program from "./command-line";
 import { Module, Result } from "./processor";
 import { RestEndpointMethodTypes } from "@octokit/rest";
+import ubiquityLogger from "../helpers/logger";
```

Comment:
```suggestion
import logger from "../helpers/logger";
```

I think its redundant to include Ubiquity as a variable identifier in these modules?

Reply:
It's to avoid a name clash with the one that is sent to the permits module, I can rename it.